pkg/cmd/admin/registry/registry.go

```diff
@@ -299,6 +303,9 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	env := app.Environment{}
 	env.Add(secretEnv)
 
+	if cfg.EnforceQuota {
+		env[server.EnforceQuotaEnvVar] = "true"
```

Comment:
Doing this means that oadm and oc have the entire docker registry codebase included, which is inappropriate.